datenguidepy/output_transformer.py

```diff
@@ -171,7 +171,7 @@ def _prefix_frame_cols(
     @staticmethod
     def _join_statistic_results(
         statistic_results: List[pd.DataFrame], statistic_names: List[str]
-    ) -> pd.DataFrame:
+    ) -> tuple:
```

Comment:
@enryH 
Same editor was crying wolf, and I accepted it's truth

Reply:
But it should return a DataFrame? @KonradUdoHannes

Reply:
From the function itself, which hasn't changed in a while tuple appears correct. I wonder how that slipped through mypy in the first place, given that we all use pre-commit hooks.
Ideally we are even more specific with the types using `Tuple`, `List`, etc. from the `typing` module as one can specify data types for content in these. For instance `Tuple[pdDataFrame,[List[str]]`.